snapcraft/manifest.txt

```diff
@@ -1,110 +1,21 @@
-adduser
-apt
-apt-utils
 base-files
 base-passwd
 bash
 bsdutils
-coreutils
 dash
-debconf
-debconf-i18n
-debianutils
-diffutils
-dmsetup
-dpkg
```

Comment:
On Tue, 2015-11-03 at 16:52 -0800, sergiusens wrote:
> In snapcraft/manifest.txt:
> >  dash
> > -debconf
> > -debconf-i18n
> > -debianutils
> > -diffutils
> > -dmsetup
> > -dpkg
> do we need this included in snaps?
I was thinking for people who may use things like dpkg-archiecture to
setup an environment variable. I'd like to find a way to not include it
unless they're using it explicitly, but I don't see a way to do that.
Perhaps we could filter it by default?